connectd/connectd.c

```diff
@@ -649,6 +649,11 @@ static void destroy_io_conn(struct io_conn *conn, struct connecting *connect)
 		errstr = "peer closed connection";
 		if (streq(connect->connstate, "Cryptographic handshake"))
 			errstr = "peer closed connection (wrong key?)";
+	} else {
+		if (connect->addrs[connect->addrnum].u.wireaddr.type == ADDR_TYPE_TOR_V3) {
+			errstr = tal_fmt(tmpctx, "%s. Hint: check debuglog and can your tor version handle a V3 onion address?",
```

Comment:
A bit ugly to do it here\idk/
Why not pass 'connect->errors' to tor.c and add the error string from debug  there  
:woman_shrugging: ???

Reply:
hmm, reviewed  my own code, seam to me my quantum Grover time body split worked.
Anyway I like this, so i will change the code and add an error_add function.
thx to my other self for review. :+1: